scripts/podcast-feed-to-content.py

```diff
@@ -422,14 +422,21 @@ def create_redirects(file_to_parse, path_md_files, redirect_prefix):
         logging.info(f"Adding redirect for episode {episode_number}")
 
         episode_file = episode.removesuffix(".md")
-        new_redirect = {
+        new_redirect_shortlink = {
             "from": f"/episodes/{episode_number}",
             "to": f"/podcast/episode/{episode_file}?pk_campaign=shortlink",
             "status": 301,
             "force": True,
         }
+        parsed_toml['redirects'].append(new_redirect_shortlink)
 
-        parsed_toml['redirects'].append(new_redirect)
+        new_redirect_episode_shortlink = {
+            "from": f"/ep{episode_number}",
+            "to": f"/podcast/episode/{episode_file}?pk_campaign=episode_shortlink",
```

Comment:
let's remove the campaign part completely, just checked that netlify forwards the query params as well. so if we don't specify anything here in the netlify.toml we can dynamically decide what param to use, e.g:
- https://engineeringkiosk.dev/ep5?pkn=shownotes
- https://engineeringkiosk.dev/ep5?pkn=twit_init
- etc....

Reply:
Fixed in https://github.com/EngineeringKiosk/webpage/pull/216/commits/e8504eb81820994d5ed93fb83c6dad51ee820b3e